### PR TITLE
Merge README.pacemaker.md into README.md

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by tboerger** [comment]
:+1:

File: README.pacemaker.md [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-[![Build Status](https://travis-ci.org/crowbar/barclamp-pacemaker.svg?branch=master)](https://travis-ci.org/crowbar/barclamp-pacemaker)
-[![Code Climate](https://codeclimate.com/github/crowbar/barclamp-pacemaker/badges/gpa.svg)](https://codeclimate.com/github/crowbar/barclamp-pacemaker)
-[![Test Coverage](https://codeclimate.com/github/crowbar/barclamp-pacemaker/badges/coverage.svg)](https://codeclimate.com/github/crowbar/barclamp-pacemaker)
-[![Dependency Status](https://gemnasium.com/crowbar/barclamp-pacemaker.svg)](https://gemnasium.com/crowbar/barclamp-pacemaker)
-[![Join the chat at https://gitter.im/crowbar/crowbar](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/crowbar/crowbar)
-
-Information for this barclamp is maintained on the [Crowbar Framework Wiki](https://github.com/crowbar/crowbar/wiki)
-
-This is NOT usable at the moment! Don't even try unless you want to help with the development effort! (Help is of
-course very welcome - contact the Crowbar mailing list.)
```
Comment (MaximilianMeister) [inline README.pacemaker.md]: so i guess this notice is outdated, that's why you didn't copy it over right?

Comment (rsalevsky) [inline README.pacemaker.md]: Right, I think this was some moths ago.

Comment (MaximilianMeister) [inline README.pacemaker.md]: okay

**Pull request merged**